[Bug]: Error sending packet ... ReferenceError: SharedArrayBuffer is not defined

### Before submitting

- [x] I have searched existing issues to make sure this bug hasn't already been reported
- [x] I have updated to the latest version of the software to verify the issue still exists
- [x] I have cleared cache/cookies/storage or tried in a private/incognito window (if applicable)

### Hardware

Rak4631

### Connection Type

Bluetooth

### Local or Hosted

https://client.meshtastic.org

### Firmware Version

2.5.20

### Operating System

macos 15.3.1

### Browser

Chrome Version 134.0.6998.44 (Official Build) (arm64)

### Expected Behavior

Bluetooth connections work without error

### Actual Behavior

I'm getting a console error when trying to connect to nodes using bluetooth.

The result is the UI not loading and being unable to use the node in the web client. 

### Steps to Reproduce

1. Click New Connection
2. Click Bluetooth
3. Click New Device
4. Select the device from the browser modal and click Pair
5. Click on the new UI button for that device


### Relevant console output

```Shell
BLE.tsx:29 Error sending packet 865484793 ReferenceError: SharedArrayBuffer is not defined
    at typedArrayToBuffer (@meshtastic_js.js?v=23ded40c:3467:31)
    at BleConnection.writeToRadio (@meshtastic_js.js?v=23ded40c:4970:81)
    at @meshtastic_js.js?v=23ded40c:4097:18
    at Queue.processQueue (@meshtastic_js.js?v=23ded40c:3559:17)
    at async BleConnection.sendRaw (@meshtastic_js.js?v=23ded40c:4096:5)
await in connect		
onConnect	@	BLE.tsx:29
onClick	@	BLE.tsx:46
```

### Screenshots

<img width="1710" alt="Image" src="https://github.com/user-attachments/assets/76d6ceef-5899-43a2-877e-c805e52694f5" />

### Additional Context

Using `git bisect` I've narrowed the problem down to this commit:

```
a5589e232b26732749e4385b427ec4248e27a91d is the first bad commit
commit a5589e232b26732749e4385b427ec4248e27a91d
Author: Sacha Weatherstone <sachaw100@hotmail.com>
Date:   Sat Mar 1 13:33:35 2025 +0800

    update dependencies

 .npmrc                                             |  2 +-
 package.json                                       | 40 ++++++-------
 src/App.tsx                                        | 30 +++++-----
 src/components/Dialog/DeviceNameDialog.tsx         |  3 +-
 src/components/Dialog/ImportDialog.tsx             | 29 +++++----
 src/components/Dialog/QRDialog.tsx                 | 18 ++++--
 src/components/PageComponents/Channel.tsx          | 68 +++++++++++-----------
 src/components/PageComponents/Config/Bluetooth.tsx |  8 +--
 src/components/PageComponents/Config/Device.tsx    |  3 +-
 src/components/PageComponents/Config/Display.tsx   |  3 +-
 src/components/PageComponents/Config/LoRa.tsx      |  3 +-
 src/components/PageComponents/Config/Network.tsx   | 18 +++---
 src/components/PageComponents/Config/Position.tsx  |  3 +-
 src/components/PageComponents/Config/Power.tsx     |  3 +-
 .../PageComponents/Config/Security/Security.tsx    | 16 +++--
 .../ModuleConfig/AmbientLighting.tsx               |  3 +-
 .../PageComponents/ModuleConfig/Audio.tsx          |  3 +-
 .../PageComponents/ModuleConfig/CannedMessage.tsx  | 18 +++---
 .../ModuleConfig/DetectionSensor.tsx               |  3 +-
 .../ModuleConfig/ExternalNotification.tsx          |  3 +-
 .../PageComponents/ModuleConfig/MQTT.tsx           | 63 ++++++++++----------
 .../PageComponents/ModuleConfig/NeighborInfo.tsx   |  3 +-
 .../PageComponents/ModuleConfig/Paxcounter.tsx     |  3 +-
 .../PageComponents/ModuleConfig/RangeTest.tsx      |  3 +-
 .../PageComponents/ModuleConfig/Serial.tsx         |  3 +-
 .../PageComponents/ModuleConfig/StoreForward.tsx   |  3 +-
 .../PageComponents/ModuleConfig/Telemetry.tsx      |  3 +-
 src/core/stores/deviceStore.ts                     | 51 +++++++---------
 src/pages/Map/index.tsx                            | 29 ++++-----
 tsconfig.json                                      |  4 +-
 30 files changed, 238 insertions(+), 204 deletions(-)
```

--- a/src/components/PageComponents/Connect/BLE.tsx
+++ b/src/components/PageComponents/Connect/BLE.tsx
@@ -5,7 +5,7 @@ import { useAppStore } from "@core/stores/appStore.ts";
 import { useDeviceStore } from "@core/stores/deviceStore.ts";
 import { subscribeAll } from "@core/subscriptions.ts";
 import { randId } from "@core/utils/randId.ts";
-import { BleConnection, Constants } from "@meshtastic/js";
+import { BleConnection, ServiceUuid } from "@meshtastic/js";
 import { useCallback, useEffect, useState } from "react";
 
 export const BLE = ({ closeDialog }: TabElementProps) => {
@@ -58,7 +58,7 @@ export const BLE = ({ closeDialog }: TabElementProps) => {
         onClick={async () => {
           await navigator.bluetooth
             .requestDevice({
-              filters: [{ services: [Constants.ServiceUuid] }],
+              filters: [{ services: [ServiceUuid] }],
             })
             .then((device) => {
               const exists = bleDevices.findIndex((d) => d.id === device.id);

--- a/vercel.json
+++ b/vercel.json
@@ -1,1 +1,18 @@
-{ "github": { "silent": true } }
+{
+  "github": { "silent": true },
+  "headers": [
+    {
+      "source": "/",
+      "headers": [
+        {
+          "key": "Cross-Origin-Embedder-Policy",
+          "value": "require-corp"
+        },
+        {
+          "key": "Cross-Origin-Opener-Policy",
+          "value": "same-origin"
+        }
+      ]
+    }
+  ]
+}

--- a/vite.config.ts
+++ b/vite.config.ts
@@ -41,7 +41,11 @@ export default defineConfig({
     },
   },
   server: {
-    port: 3000
+    port: 3000,
+    headers: {
+      'Cross-Origin-Opener-Policy': 'same-origin',
+      'Cross-Origin-Embedder-Policy': 'require-corp',
+    }
   },
   optimizeDeps: {
     exclude: ['react-scan']

